Update two of my previous papers

jianwang-mpi opened the pull request:
Update two of my previous papers: 
1. add detailed information about the paper "Estimating Egocentric 3D Human Pose in the Wild with External Weak Supervision"
2. add paper: "Estimating Egocentric 3D Human Pose in Global Space"

jianwang-mpi committed:
Update two of my previous papers

Update two of my previous papers: 
1. add detailed information about the paper "Estimating Egocentric 3D Human Pose in the Wild with External Weak Supervision"
2. add paper: "Estimating Egocentric 3D Human Pose in Global Space"

Sid2697 commented:
Hello @yt4766269,

Thank you for your interest in the repository and updating the information. Before I merge your PR with the base branch, can you please update your paper's information here too? https://github.com/Sid2697/awesome-egocentric-vision/blob/9c7a4e0b916aeb7edad5a6e3cf17cb5c2114db11/README.md?plain=1#L305

Regards,
Siddhant Bansal

jianwang-mpi committed:
Update details of my paper under "CVPR"

Update details of paper "Estimating Egocentric 3D Human Pose in the Wild with External Weak Supervision" under "CVPR"

jianwang-mpi commented:
Hello @Sid2697 ,

I have updated my paper's information under "CVPR". Sorry for forgetting this.

Best,

Jian
